frontend/src/entities/towers/Tower.js

```diff
@@ -1,6 +1,7 @@
 import Entity from "../Entity";
 
 export const Status = {
+    PLANNING: Symbol("planning"),
```

Comment:
So in the github issue I didn't follow at first why you wanted a new state, but I guess you wanted to distinguish between towers built and not yet built in the current build phase? That way you could make the built ones selectable.

Reply:
Yep, actually that should be working on this branch. However, you kind of need to deselect the tower you're buying, or it will build that tower on the last tile that you passed before selecting the `PENDING` tower